### DISCUSSION TITLE
Linux / Miningcore: command not found

### DISCUSSION THREAD

**Discussion by bdk773** [coinfoundry/miningcore#913] [Q&A] [answered]
I've tried to rebuild the repo more than once but I'm keep getting this error. Any advise on why? We can clearly see the program is there:

https://pastebin.com/uxKP8bHQ

![image](https://user-images.githubusercontent.com/87677261/128267909-b227dff3-88a3-40ed-ab32-c28e36497290.png)

Thank you in advance.

**Comment by oliverw** [comment, marked as answer]
Linux ain't Windows, thus the current directory is normally not part of `PATH`. Enter `./Miningcore`.

**Reply by bdk773** [reply]
My brain is not functioning...! thank You 🙏
